hw/opentitan/ot_i2c.c

```diff
@@ -967,9 +1092,27 @@ static void ot_i2c_dj_write(void *opaque, hwaddr addr, uint64_t val64,
         s->regs[reg] = val32;
         s->check_timings = true;
         break;
+    case R_CONTROLLER_EVENTS:
+        val32 &= CONTROLLER_EVENTS_RW1C_MASK;
+        s->regs[reg] &= ~val32;
```

Comment:
Minor nit, just convention in the OpenTitan QEMU code: can you add a comment like:

```suggestion
        s->regs[reg] &= ~val32; /* RW1C */
```

on these RW1C fields so it is clear to someone casually browsing why the write is implemented this way?